Makefile.in

```diff
@@ -14,7 +14,9 @@ CPU = @target_cpu@
 TARGET = @target_alias@
 SYMBOL_PREFIX = @SYMBOL_PREFIX@
 CFLAGS = @CFLAGS@
-INCLUDES = -I$(srcdir) @bfd_include_dir@ @binutils_include_dir@ @zlib_include_dir@
+INCLUDES = -I$(srcdir) -Ibfd-headers @zlib_include_dir@
+BFD_INCLUDE_DIR = $(shell cd "@bfd_include_dir@" && pwd)
```

Comment:
I am quoting your comment:

[[[[
you can do this instead in the shell code:

`bfddir=$(cd "$(BFD_INCLUDE_DIR)"; pwd)`

then you'll have access to $$bfddir everywhere.

alternatively, update the configure.ac code to do that:

`bfd_include_dir="$(cd "$ac_bfd_include_dir"; pwd)"`
]]]]

If you preferred one way and not the other, you shouldn't have placed the *not recommended* way as the first suggestion.

I am not making another patch for this, sorry. This PR takes much longer than it should have had.